src/node_kernels/TKESSTNodeKernel.C

```diff
@@ -19,29 +19,31 @@
 namespace sierra {
 namespace nalu {
 
-TKESSTNodeKernel::TKESSTNodeKernel(const stk::mesh::MetaData& meta)
-  : NGPNodeKernel<TKESSTNodeKernel>(),
-    tkeID_(get_field_ordinal(meta, "turbulent_ke")),
-    sdrID_(get_field_ordinal(meta, "specific_dissipation_rate")),
-    densityID_(get_field_ordinal(meta, "density")),
-    tviscID_(get_field_ordinal(meta, "turbulent_viscosity")),
-    dudxID_(get_field_ordinal(meta, "dudx")),
-    dualNodalVolumeID_(get_field_ordinal(meta, "dual_nodal_volume")),
-    nDim_(meta.spatial_dimension())
+TKESSTNodeKernel::TKESSTNodeKernel(
+  const stk::mesh::MetaData& meta,
+  const FieldManager& manager,
+  stk::mesh::Part* part)
+  : NGPNodeKernel<TKESSTNodeKernel>(), nDim_(meta.spatial_dimension())
 {
+  manager.register_field("turbulent_ke", part);
+  manager.register_field("specific_dissipation_rate", part);
+  manager.register_field("density", part);
+  manager.register_field("turbulent_viscosity", part);
+  manager.register_field("dudx", part);
```

Comment:
I wonder if we should have some central location where field-names are associated with an enum or something. Because having string-literals scattered around in various kernels could be an opportunity for errors.